packages/features/tips/Tips.tsx

```diff
@@ -175,3 +172,5 @@ export default function Tips() {
     </div>
   );
 }
+
+export default memo(Tips);
```

Comment:
This just renders from internal data - no point re-rendering on every parent change (since this was in shell it was re-rendering a lot)